src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts\Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts\Avenir-Book.ttf);
+}
+
+:root {
```

Comment:
Do not raise the specification using this pseudo-class. Assign a class to the tag and apply the style.